File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    // Release 0
    let answer = ''

    for (let i = 0; i < arrayToSearchThrough.length; i ++) {
        if (arrayToSearchThrough[i] == valueToFind) {
            if (!answer) {
                answer = i
            }
        }
    }
    if (answer) {
        return answer
    } else {
        return undefined
    }
};

exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    let answer = []

    for (let i = 0; i < arrayToSearchThrough.length; i ++) {
        if (arrayToSearchThrough[i] == valueToFind) {
            answer.push(i)
        }
    }
    return answer
}

File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    # your code here
    if value_to_find in array_to_search_through:
        return array_to_search_through.index(value_to_find)


def global_linear_search(value_to_find, array_to_search_through):
    answer = []

    for i in range(len(array_to_search_through)):
        if value_to_find == array_to_search_through[i]:
            answer.append(i)
    return answer